JASP-Desktop/po/jasp_nl.po

```diff
@@ -16078,10 +15845,2443 @@ msgctxt "RibbonModel|"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Absolute"
+msgstr "Absoluut"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Relative"
+msgstr "Relatief"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Population"
+msgstr "Populatie"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Size"
+msgstr "Grootte"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Value"
+msgstr "Waarde"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Audit Risk"
+msgstr "Audit Risico"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Confidence"
+msgstr "Betrouwbaarheid"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Record ID's"
+msgstr "Transactie ID's"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Audit Result"
+msgstr "Auditresultaat"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Book Values <i>(required)</i>"
+msgstr "Boekwaarden <i>(verplicht)</i>"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Book Values <i>(optional)</i>"
+msgstr "Boekwaarden <i>(optioneel)</i>"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Selection Counter <i>(optional)</i>"
+msgstr "Selectieteller <i>(optioneel)</i>"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Annotation Method"
+msgstr "Annotatiemethode"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Audit values"
+msgstr "Audit waarden"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Correct / Incorrect"
+msgstr "Correct / Incorrect"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Use summary statistics"
+msgstr "Gebruik samenvattende statistieken"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Sample size"
+msgstr "Steekproefgrootte"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Found errors"
+msgstr "Gevonden fouten"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Advanced Options"
+msgstr "Geavanceerde Opties"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Estimation Method"
+msgstr "Schattingsmethode"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Stringer"
+msgstr "Stringer"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "LTA adjustment"
+msgstr "LTA bijstelling"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Direct"
+msgstr "Direct"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Difference"
+msgstr "Verschil"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Ratio"
+msgstr "Ratio"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Regression"
+msgstr "Regressie"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Binomial"
+msgstr "Binomiaal"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Poisson"
+msgstr "Poisson"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Hypergeometric"
+msgstr "Hypergeometrisch"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Inherent Risk"
+msgstr "Inherent Risico"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "High"
+msgstr "Hoog"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Medium"
+msgstr "Middel"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Low"
+msgstr "Laag"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Custom"
+msgstr "Aangepast"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Control Risk"
+msgstr "Controlerisico"
```

Comment:
Controlerisico vs Inherent Risico

Reply:
Eigenlijk stel ik voor dat `Controlerisico` vertaald word als `Intern Beheersingsrisico` overal.

Reply:
Okay ik verander het in mijn versie. Ik probeer zo min mogelijk te veranderen.